### ISSUE TITLE
[BUG] Specviz: Application not opening!

### ISSUE THREAD

**Issue by atharvabagul2000** [spacetelescope/jdaviz#1676]
**Describe the bug**
I have been using SpecViz for quite a while, today, I reloaded and had to run my code, but the SpecViz application is not opening.

It is getting imported and showing no error, but the application is not visible. I am getting the following response when I load the application on Jupyter Notebook. I also tried !pip install jdaviz --upgrade and after restarting the kernel I ran SpecViz. It is still showing the same message instead of loading the application.

Application(config='specviz', events=['call_viewer_method', 'close_snackbar_message', 'data_item_remove', 'dat…

**Expected behavior**
When I run the following:

specviz = Specviz()
specviz.load_spectrum(...)
specviz.app

It should show me the application window in the output section of Jupyter cell.

**Desktop (please complete the following information):**
 - MacOS (latest updated)
 - Browser: Safari
 - Jupyter: notebook 6.4.12

INC0182407

**Comment by pllim** [comment]
Do you still see this problem if you create a new conda environment and only install Jdaviz there?

Do you see any errors in the browser console?

**Comment by atharvabagul2000** [comment]
I was not getting this issue till sometime back. This problem just occurred a few days back. I didn't see any issues in the browser console.

**Comment by pllim** [comment]
Sometimes some upstream update would cause incompatibility that we cannot control. If the problem go away with a new environment, then just use the new environment. 🤞

**Comment by atharvabagul2000** [comment]
> Sometimes some upstream update would cause incompatibility that we cannot control. If the problem go away with a new environment, then just use the new environment. 🤞

I am not getting it either in a new environment I created. It is still showing the same issue.

**Comment by pllim** [comment]
We just released v3.0.1 yesterday. Can you please upgrade again and see? Thanks!

**Comment by pllim** [comment]
I am not getting enough information here to help you debug. If you wish to Slack call and share your screen, we can try that on Astropy Slack: http://joinslack.astropy.org/

**Comment by atharvabagul2000** [comment]
Hello,I am available for a short meeting at about 11:00am Baltimore time. Let me know if that works for you!Thank you!Sent from my iPhoneOn 11-Oct-2022, at 6:31 PM, P. L. Lim ***@***.***> wrote:﻿
I am not getting enough information here to help you debug. If you wish to Slack call and share your screen, we can try that on Astropy Slack: http://joinslack.astropy.org/

**Comment by pllim** [comment]
Alas, that is not good for me, but please direct message me (pllim) in that Slack workspace and reference this issue. We can go from there. Thanks!

**Comment by pllim** [comment]
Update 2022-10-12: We created another fresh enviroment with Python 3.9 and did a `pip install jdaviz --upgrade` (v3.0.1). We did have to make sure notebook was pointing to the correct conda env. The research notebook in question took a long time to load but investigating that is out of scope. We were able to make Jdaviz application display in a simple notebook but:

* Some Jdaviz import has a noticeable lag (not sure if related or is something else). `import numpy` was instantaneous.
* Jdaviz application does not display (as reported above) when it is run as part of the heavyweight research notebook.

So, something in that research notebook is messing up Jdaviz widget application rendering but I don't know what.

Next step: User would provide a more minimal version of the research notebook, just enough to reproduce the problem. Send that along with necessary data to help call. Dev(s) will try to reproduce the problem.

User indicated that separating out just Jdaviz parts into a separate notebook was not a viable solution for the research workflow. User also wants @PatrickOgle to be in the loop on this issue.
